packages/simple-schema/lib/content-assist/element-name.js

```diff
@@ -11,7 +13,22 @@ const { difference, map, filter } = require("lodash");
  * @returns {CompletionSuggestion[]}
  */
 function elementNameCompletion(elementNode, xssElement, prefix = "") {
-  const allPossibleSuggestions = map(xssElement.elements, _ => _.name);
+  const match = prefix.match(NAMESPACE_PATTERN);
+  if (!match) {
+    return [];
+  }
+  const namespacePrefix = match[1];
+  const namespace = find(

```

Comment:
maybe a better name for `namespace`? 
very similar to `_.namespace`

One is from the prefix, the other from the element, can we distinguish them better?